src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SingleIconSelect.js

```diff
@@ -0,0 +1,129 @@
+// @flow
+import React, {Fragment} from 'react';
+import {observer} from 'mobx-react';
+import {action, observable, computed} from 'mobx';
+import SingleItemSelection from '../../../components/SingleItemSelection';
+import {translate} from '../../../utils/Translator';
+import SingleListOverlay from '../../SingleListOverlay';
+import userStore from '../../../stores/userStore';
+import singleIconSelectStyle from '../../../components/SingleIconSelect/singleIconSelect.scss';
+import type {FieldTypeProps} from '../../../types';
+import type {IObservableValue} from 'mobx/lib/mobx';
+
+type Props = FieldTypeProps<?string>
+
+@observer
+export default class SingleIconSelect extends React.Component<Props> {
```

Comment:
as this is implemented using a list overlay now, i guess it would be more consistent to call it `SingleIconSelection` (same for the name of the field type and content type). but i am not sure if you already talked about this and if there are good reasons to keep `select`

Reply:
also, we should add basic test cases for the component before merging it

Reply:
@alexander-schranz @niklasnatter  we talked about that but i think this was before the implementation with lists... should we use select or selection? Also i would appreciate some help with the test cases...

Reply:
you dont need to worry about the tests too much. if everything else works, i can add the test cases 🙂

Reply:
@alexander-schranz what about the naming? select or selection?